src/mono/wasm/Wasm.Build.Tests/BuildPublishTests.cs

```diff
@@ -43,7 +43,7 @@ public void Wasm_CannotAOT_InDebug(BuildArgs buildArgs, RunHost _, string id)
             Assert.Contains("AOT is not supported in debug configuration", buildOutput);
         }
 
-        [Theory]
+        [Theory, TestCategory("no-fingerprinting")]
```

Comment:
please add comment why this test is not compatible. Probably on all of them

Reply:
It's the other way around. This is a semi-randomly picked test to be run even without fingerprinting

Reply:
Maybe rename it to `test-also-without-fingerprint` or something similar?